mcs/class/System.Messaging/System.Messaging/MessageQueue.cs

```diff
@@ -32,7 +32,9 @@
 using System;
 using System.Collections;
 using System.ComponentModel;
+#if !NO_SYSTEM_DRAWING_DEPENDENCY
 using System.Drawing;
+#endif
```

Comment:
It's from
[TypeConverter (typeof(SizeConverter))]

https://msdn.microsoft.com/en-us/library/system.drawing.sizeconverter(v=vs.110).aspx